core/src/main/scala/com/stratio/crossdata/launcher/SparkJobLauncher.scala

```diff
@@ -37,8 +38,9 @@ object SparkJobLauncher extends SparkLoggerComponent with CrossdataSerializer {
   def getSparkStreamingJob(crossdataConfig: Config, streamingCatalog: XDStreamingCatalog, ephemeralTableName: String)
                           (implicit executionContext: ExecutionContext): Try[SparkJob] = Try {
     val streamingConfig = crossdataConfig.getConfig(StreamingConfPath)
+    val launcherConfig=crossdataConfig.getConfig(LauncherKey)
     val sparkHome =
-      Properties.envOrNone("SPARK_HOME").orElse(Try(streamingConfig.getString(SparkHomeKey)).toOption).getOrElse(
+      Properties.envOrNone("SPARK_HOME").orElse(Try(launcherConfig.getString(SparkHomeKey)).toOption).getOrElse(
```

Comment:
I suppose that there is only one Spark installation that is defined in tehe SPARK_HOME variable, and it can be used for everything